packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -143,6 +144,15 @@ export class ApmConfiguration {
       config.active = true;
     }
 
+    if (process.env.ELASTIC_APM_KIBANA_FRONTEND_ACTIVE === 'false') {
+      // @ts-ignore
+      config.servicesOverrides = {
+        'kibana-frontend': {
+          active: false,
+        },
+      };
+    }
```

Comment:
I'm not very familiar with the `servicesOverrides` config, but because this is a map, would it be safer to just set 'kibana-frontend` instead of clobbering the entire `servicesOverrides` object?

Reply:
I pushed https://github.com/elastic/kibana/pull/179062/commits/8cfc530b29174c9d078fc124bf89fd046527dcfb